cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,145 @@
+        - prefices: list of one or more str prefices to match (e.g. ["transformers", "torch"]). Optional.
+          Default is `[""]` to match all active loggers.
+          The match is a case-sensitive `module_name.startswith(prefix)`
+    """
+    prefix_re = re.compile(fr'^(?:{ "|".join(prefices) })')
+    for name in logging.root.manager.loggerDict:
+        if re.match(prefix_re, name):
+            logging.getLogger(name).setLevel(level)
+
+class MultiOutputBert(nn.Module):
+    def __init__(self, num_labels):
+        super(MultiOutputBert, self).__init__()
```

Ruff

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)